### PR TITLE
Cover letter and doc updates

### PR DESCRIPTION
**Pull request by sonals**

### CODE REVIEW

File: src/Documentation/fpga/xrt.rst [hunk 1]
```diff
@@ -251,17 +264,18 @@ Driver Interfaces
...
+Ioctls exposed by xmgmt driver to user space are enumerated in the following
+table:
 
-== ===================== ============================= ===========================
+== ===================== ============================ ==========================
 #  Functionality         ioctl request code            data format
-== ===================== ============================= ===========================
+== ===================== ============================ ==========================
 1  FPGA image download   XMGMT_IOCICAPDOWNLOAD_AXLF    xmgmt_ioc_bitstream_axlf
 2  CL frequency scaling  XMGMT_IOCFREQSCALE            xmgmt_ioc_freqscaling
```
Comment (maxzhen) [inline src/Documentation/fpga/xrt.rst:274]: We actually did not implement XMGMT_IOCFREQSCALE. Wondering if we should remove this from our doc and header?

Comment (sonals) [reply src/Documentation/fpga/xrt.rst:274]: Point taken, will update the PR

Comment (sonals) [reply src/Documentation/fpga/xrt.rst]: done

Comment (maxzhen) [reply src/Documentation/fpga/xrt.rst]: Thanks. I'll file a PR to remove it from our header now.

**Review by maxzhen — APPROVED** [review]

**Pull request merged**